cluster/userdata-master.yaml

```diff
@@ -757,6 +765,90 @@ write_files:
               - kubectl
               - proxy
 
+  - path: /srv/kubernetes/manifests/prometheus-node-exporter-de.yaml
+    content: |
+      apiVersion: extensions/v1beta1
+      kind: DaemonSet
+      metadata:
+        name: prometheus-node-exporter
+        labels:
+          app: prometheus
+          component: node-exporter
+        namespace: kube-system
+      spec:
+        template:
+          metadata:
+            name: prometheus-node-exporter
+            labels:
+              app: prometheus
+              component: node-exporter
+          spec:
+            containers:
+            - image: prom/node-exporter:0.12.0
+              name: prometheus-node-exporter
+              ports:
+              - name: prom-node-exp
+                containerPort: 9100
+                hostPort: 9100
+            hostNetwork: true
+            hostPID: true
+  - path: /srv/kubernetes/manifests/prometheus-node-exporter-svc.yaml
```

Comment:
There should be a newline between the two file definitions. Just for consistency :)